tests/db-sqlalchemy/accounts_manager_test.py

```diff
@@ -17,24 +17,18 @@ def accounts_manager(
 
 def test_accounts_manager(accounts_manager: AccountsManager):
     address = "0x9F0e84243496AcFB3Cd99D02eA59673c05901501"
-    balance = 100
-    accounts_manager.fund_account(address, balance)
+    accounts_manager.create_new_account_with_address(address)
```

Comment:
could you test more methods from the AccountsManager here?

Reply:
Done! thanks to cursor :D